packages/framework/fluid-static/package.json

```diff
@@ -140,6 +140,34 @@
 			"TypeAliasDeclaration_Myself": {
 				"forwardCompat": false,
 				"backCompat": false
+			},
+			"InterfaceDeclaration_IFluidContainer": {
```

Comment:
The API breakages seem to be (for the ones I looked at in detail anyway) improved type safety catching more incompatibilities from the IFLuidHandle refactor which were previously missed.